tee-worker/service/src/main_impl.rs

```diff
@@ -241,9 +241,9 @@ pub(crate) fn main() {
 	} else if matches.is_present("shielding-key") {
 		setup::generate_shielding_key_file(enclave.as_ref());
 	} else if matches.is_present("signing-key") {
-		setup::generate_signing_key_file(enclave.as_ref());
 		let tee_accountid = enclave_account(enclave.as_ref());
-		println!("Enclave signing account: {:}", &tee_accountid.to_ss58check());
+		println!("Enclave account: {:}", &tee_accountid.to_ss58check());
+		setup::generate_signing_key_file(enclave.as_ref());
```

Comment:
Is this change intentional? 🤔

Reply:
No, sorry. It must come from the merging.

I reverted it and adjusted `bitacross-worker/srvice/src/main_impl.rs`